### PR TITLE
Adds Hornet weapon

### PR DESCRIPTION
**Pull request by monochromatism**
## About The Pull Request

Adds the Hornet rifle, E.G.O. weapon of Queen Bee. Should be fully functional. (Please ignore the amount of commits, I'm stupid.)

## Why It's Good For The Game

Queen Bee currently only has its armor. The game needs good ranged red damage that DOESN'T kill everyone in the vicinity of the target.

## Changelog
:cl:
add: Added Hornet rifle
:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/obj/item/gun/ego_gun/hornet
+	name = "hornet"
+	desc = "The kingdom needed to stay prosperous, and more bees were required for that task.\
+	The projectiles relive the legacy of the kingdom as they travel toward the target."
+	icon_state = "hornet"
+	inhand_icon_state = "hornet"
+	w_class = WEIGHT_CLASS_NORMAL
+	ammo_type = /obj/item/ammo_casing/caseless/ego_hornet
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/ego_gun/waw.dm:8]: Make it unable to be used with one hand like the fourth match is, also make it bulky (can still fit on your belt

**Comment by Kitsunemitsu** [comment]
Okay!
we tested this today.
1) You cannot but this in the ego console
2) The right and left sprites are flipped
3) It has no stat requirement
4) It needs to be a bulky weapon that fits on belt

**Comment by monochromatism** [comment]
I think that should cover all the issues? I hope??

**Comment by EgorDinamit** [comment]
> I think that should cover all the issues? I hope??

Wild merge conflicts appeared!

File: code/modules/mob/living/simple_animal/abnormality/waw/queen_bee.dm [hunk 2]
```diff
@@ -22,6 +22,7 @@
 	work_damage_type = RED_DAMAGE
 
 	ego_list = list(
+		/datum/ego_datum/weapon/hornet
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/queen_bee.dm:25]:
```suggestion
		/datum/ego_datum/weapon/hornet,
```

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 3]
```diff
@@ -14,3 +14,17 @@
 							FORTITUDE_ATTRIBUTE = 40,
 							JUSTICE_ATTRIBUTE = 60
 							)
+/obj/item/ego_weapon/hornet
```
Comment (EgorDinamit) [inline code/modules/projectiles/guns/ego_gun/waw.dm:17]: Why is it under ego_weapon? It needs to go to ego_gun.

**Comment by monochromatism** [comment]
I'm just going to start over. This is unrecoverable. Sorry to waste time! Hopefully I won't fuck up this badly again.

**Pull request closed without merging**